conda_build/build.py

```diff
@@ -888,6 +888,8 @@ def build(m, post=None, need_source_download=True, need_reparse_in_env=False, bu
         # installed files at packaging-time.
         host_ms_deps = None
         build_ms_deps = None
+        test_ms_deps = m.get_value('test/requires', [])
```

Comment:
You should add the runtime dependencies to this also.  Test time deps are actually run + test deps.